Backend/Core/Infrastructure.Business/Users/UserService.cs

```diff
@@ -18,29 +18,30 @@ public UserService(IUserRepository<T> repository, IIdentityUserService identityU
 
         public IIdentityUserService IdentityUserService { get; }
 
-        public abstract string[] GetSpecificRoles();
+        public abstract Task<string[]> GetSpecificRoles();
 
         public async Task<T> Create(string firstName, string lastName, string phoneNumber)
```

Comment:
I think "Create" is not suitable name for this the method. We don't create new entity, we only assign some properties, like firstName, lastName, roles.